autoload/ledger.vim

```diff
@@ -393,6 +393,18 @@ function! ledger#align_amount_at_cursor()
   endif
 endf!
 
+function! ledger#autocomplete_and_align()
+  if pumvisible()
+    return "\<c-n>"
+    " See http://stackoverflow.com/questions/23323747/vim-vimscript-get-exact-character-under-the-cursor
```

Comment:
One hypothetical problem with code taken from stack overflow is that its license (cc by-sa 3.0) is not compatible with the GPL. On the other hand the `matchstr` / `\%c` workaround is commonly used and likely did not originate there. So I think this should be okay.